libmscore/read206.cpp

```diff
@@ -138,7 +138,7 @@ struct StyleVal2 {
       { Sid::measureSpacing,              QVariant(1.2) },
       { Sid::staffLineWidth,              Spatium(0.08) },      // 0.09375
       { Sid::ledgerLineWidth,             Spatium(0.16) },     // 0.1875
-      { Sid::ledgerLineLength,            Spatium(.6) },     // notehead width + this value
+      { Sid::ledgerLineLength,            Spatium(0.76) },     // notehead width + this value
```

Comment:
Add width to length to compensate for `Qt::FlatCap` not needed extra space, so by default the lines end up with the same length, here for importing 2.x scores.